experimental/framework/frs-client/src/interfaces.ts

```diff
@@ -42,13 +42,14 @@ export interface FrsContainerServices {
  * Since Frs provides user names for all of its members, we extend the IMember interface to include
  * this service-specific value. It will be returned for all audience members connected to Frs.
  */
-export interface FrsMember extends IMember {
+export interface FrsMember<T> extends IMember {
     userName: string;
+    additionalDetails: T;
 }
 
 export interface FrsResources {
     fluidContainer: FluidContainer;
     containerServices: FrsContainerServices;
 }
 
-export type IFrsAudience = IServiceAudience<FrsMember>;
+export type IFrsAudience = IServiceAudience<FrsMember<any>>;
```

Comment:
```suggestion
export type IFrsAudience<T = any> = IServiceAudience<FrsMember<T>>;
```
This would allow the objects returned by the audience to also be typed according to the generic the dev passes in